src/docs/development/tools/hot-reload.md

```diff
@@ -69,7 +69,7 @@ Dart code to keep using hot reload.
 
 ## Previous state is combined with new code
 
-Flutter's hot reload feature, sometimes described as _stateful hot reload_,
+Flutter's hot reload feature, sometimes described as _Stateful Hot Reload_,
```

Comment:
This is one of those sort of tricky situations. The guideline says to use Stateful Hot Reload when referencing the official Flutter feature. But this sentence says, "sometimes described as..." which makes it a more colloquial reference, rather than an Official Feature Reference.  I think case, lower case is fine.  (I'd be interested to know whether K agrees with this.)

Reply:
For consistency's sake, how about :

`Flutter's Stateful Hot Reload` 
and drop the `sometimes described` (as I'm not sure what value that's adding here)

Reply:
We should avoid colloquial reference per the style guide:
https://developers.google.com/style/tone

Reply:
I think they refer to "colloquial" as in idioms or slang. And, yes, that is to be avoided. But what I mean is different. It's better to use English than to always use code fonted words. Maybe I'm not describing it well.

But I agree that deleting "sometimes described as" here would make the upper case version acceptable.